### PR TITLE
[Review][ios10,UIKit] UIGraphicsRenderer

### PR DESCRIPTION
**Pull request by migueldeicaza**

### CODE REVIEW

**Comment by xamarin-release-manager** [comment]
Build success

File: src/uikit.cs [hunk 1]
```diff
@@ -5175,6 +5175,164 @@ public interface UIGestureRecognizerDelegate {
 		bool ShouldReceivePress (UIGestureRecognizer gestureRecognizer, UIPress press);
 	}
 
+	[iOS (10,0)]
+	[BaseType (typeof(NSObject))]
+	interface UIGraphicsRendererFormat : NSCopying
+	{
+		[Static]
+		[Export ("defaultFormat")]
+		UIGraphicsRendererFormat DefaultFormat ();
```
Comment (spouliot) [inline src/uikit.cs]: It should it be a getter-only property (like our `SharedInstance` are) or be prefixed with a `Get` so there's a verb in the method name.

File: src/uikit.cs [hunk 2]
```diff
@@ -5175,6 +5175,164 @@ public interface UIGestureRecognizerDelegate {
...
+		[Export ("bounds")]
+		CGRect Bounds { get; }
+	}
+
+	[iOS (10,0)]
+	[BaseType (typeof(NSObject))]
+	interface UIGraphicsRendererContext
+	{
+		[Export ("CGContext")]
+		unsafe CGContext CGContext { get; }
```
Comment (rolfbjarne) [inline src/uikit.cs]: This `unsafe` looks redundant.

File: src/uikit.cs [hunk 3]
```diff
@@ -5175,6 +5175,164 @@ public interface UIGestureRecognizerDelegate {
...
+		CGRect PdfContextBounds { get; }
+
+		[Export ("beginPage")]
+		void BeginPage ();
+
+		[Export ("beginPageWithBounds:pageInfo:")]
+		void BeginPage (CGRect bounds, NSDictionary<NSString, NSObject> pageInfo);
+
+		[Export ("setURL:forRect:")]
+		void SetURL (NSUrl url, CGRect rect);
```
Comment (rolfbjarne) [inline src/uikit.cs]: `Url` instead of `URL` maybe?

Comment (spouliot) [inline src/uikit.cs]: should be `SetUrl`

File: src/uikit.cs [hunk 4]
```diff
@@ -5175,6 +5175,164 @@ public interface UIGestureRecognizerDelegate {
...
+
+		[Export ("addDestinationWithName:atPoint:")]
+		void AddDestinationWithName (string name, CGPoint point);
```
Comment (spouliot) [inline src/uikit.cs]: I would remove the `WithName` suffix (here and next API) as the parameter name provides it.

**Comment by migueldeicaza** [comment]
Thanks for the review, updated with the feedback.

**Comment by xamarin-release-manager** [comment]
Build success

**Comment by migueldeicaza** [comment]
build

**Comment by xamarin-release-manager** [comment]
Build success

**Pull request merged**